static/src/javascripts/projects/common/modules/experiments/tests/newsletter-embed-test.js

```diff
@@ -1,4 +1,19 @@
 // @flow
+import ophan from 'ophan/ng';
+
+const trackComponentInOphan = (newsletterId: string, variant: string) => {
+    ophan.record({
+        action: 'INSERT',
+        component: {
+            componentType: 'NEWSLETTER_SUBSCRIPTION',
+            id: newsletterId
+        },
+        abTest: {
+            name: 'NewsletterEmbeds',
+            variant
+        }
+    });
+};
 
 export const newsletterEmbeds: ABTest = {
     id: 'NewsletterEmbeds',
```

Comment:
Perhaps we should update this ID to make it easy to see when we fixed the issue and restarted the test?